### ISSUE TITLE
Better cursor handling for contentEditable=false islands. 

### ISSUE THREAD

**Issue by aerych** [wordpress-mobile/WordPress-Editor-iOS#555]
We can make an element un-editable by adding the `contentEditable=false` attribute, thus preventing keyboard input inside the element.  It also blocks bold, italic, etc. actions from the format bar. 

However, it is possible for the user to still tap inside the element, placing the cursor inside an uneditable island.  Ideally we'd like to prevent this, or move the caret position to just after the element when the user begins typing.  This seems tricky as the uneditable content does not seem to respond to useful events (e.g. focus, focusin, keydown, etc.). 

Find an implement a way to reliably reposition the cursor so it is not trapped in an un-editable island.

**Comment by diegoreymendez** [comment]
@aerych - Try adding this by hand in the HTML editor:

```html
<b contentEditable="false" style="-webkit-user-select: none;">Hello there</b>
```

**Comment by aerych** [comment]
@diegoreymendez Oh nice! Thanks for the tip. That looks like its gonna do the trick :)

**Comment by diegoreymendez** [comment]
@aerych - Shall we close this or do we want to make this more specific to fix certain tags?

**Comment by aerych** [comment]
Closed in #580

**Comment by diegoreymendez** [comment]
:+1:
